.luacheckrc

```diff
@@ -0,0 +1,12 @@
+unused_args = false
+
+read_globals = {
+    "table",
+    "minetest",
+    "digiline",
+    "digilines"
```

Comment:
`"digiline"` should not be included, all references to `digiline` should be changed to `digilines`.

Also, `"table"` should be `"table.copy"`. Better to be strict with luacheck :)